quarry/utils/buffer.py

```diff
@@ -76,11 +106,13 @@ def unpack(self, fmt):
         for ``struct.unpack()``.
         """
 
-        fmt = ">"+fmt
+        fmt = ">%s" % (fmt)
```

Comment:
This can stay as `fmt = ">" + fmt`, it still looks fine stylistically and *technically* performs better. Same goes for line 212.